command/repo.go

```diff
@@ -51,12 +51,14 @@ A repository can be supplied as an argument in any of the following formats:
 }
 
 var repoCloneCmd = &cobra.Command{
-	Use:   "clone <repo>",
+	Use:   "clone <repo> [<directory>]",
 	Args:  cobra.MinimumNArgs(1),
 	Short: "Clone a repository locally",
 	Long: `Clone a GitHub repository locally.
 
-To pass 'git clone' options, separate them with '--'.`,
+To pass 'git clone' options, separate them with '--'.
+
+In order to clone to a specific directory, provide it before the '--' instead of after the other additional options.`,
```

Comment:
Hmm let's avoid adding this. It's technically true, but it makes help text harder to grok, and it should be obvious from the usage synopsis that `<directory>` should come before the `--`.

Reply:
Sounds good. Fixed in 199021d55daecb432cee438051401edf4258c161